app/views/session_answers/coronavirus_find_support/_results.html.erb

```diff
@@ -0,0 +1,46 @@
+<% content_for :title do %>
+  Information based on your answers
+<% end %>
+
+<div class="govuk-!-margin-top-3 govuk-!-margin-bottom-6">
+  <%= render "govuk_publishing_components/components/heading", {
+    text: "Information based on your answers",
+    font_size: "xl"
+  } %>
+  <a class="govuk-link" href="/coronavirus_find_support">Start again</a>
+</div>
+
+<div data-track-ec-list="Find coronavirus support">
+  <div class="govuk-width-container">
+    <main class="govuk-main-wrapper govuk-main-wrapper--l" id="main-content" role="main">
+      <% session[:coronavirus_find_support].keys.each do |group| %>
```

Comment:
If everywhere `session[:coronavirus_find_support]` appears in this partial, it was replaced with `session[flow_name]`, the partial could be shared across most (all?) flows.